src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -482,6 +482,8 @@ impl Renderer for HtmlHandlebars {
         // Copy all remaining files, avoid a recursive copy from/to the book build dir
         utils::fs::copy_files_except_ext(&src_dir, destination, true, Some(&build_dir), &["md"])?;
 
+        info!("Book has been generated into `{}`", build_dir.display());
```

Comment:
Was there a particular reason to use `build_dir` instead of `destination`? It might be confusing, since this is specifically for the HTML renderer, but this is showing the root output directory which might not be where the HTML output actually resides. Maybe something like this?

```suggestion
        info!("HTML book written to `{}`", destination.display());
```

Reply:
No particular reason, simply missed that HTML output folder might not be the same as build folder.